From: Alec Leamas <leamas.alec@gmail.com>
Date: Wed, 20 Nov 2019 15:47:38 -0500
Subject: CMakelists: move appdata.xml to /usr/share/metainfo

The former location /usr/share/appdata/opencpn.appdata.xml has been
deprecated, see
https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html

Bug: https://github.com/OpenCPN/OpenCPN/issues/1892
---
 CMakeLists.txt | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 0ea5b86..021692a 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -2175,9 +2175,11 @@ if (NOT APPLE)
       DESTINATION ${PREFIX_DATA}/icons/hicolor/scalable/apps
     )
     install(FILES data/opencpn.desktop DESTINATION ${PREFIX_DATA}/applications)
+    install(DIRECTORY DESTINATION ${PREFIX_DATA}/metainfo)
     install(
       FILES ${CMAKE_BINARY_DIR}/opencpn.appdata.xml
-      DESTINATION ${PREFIX_DATA}/appdata
+      DESTINATION ${PREFIX_DATA}/metainfo
+      RENAME opencpn.metainfo.xml
     )
     install(FILES opencpn.1 DESTINATION ${PREFIX_DATA}/man/man1)
   endif (UNIX)
